docs/hitran/hitran.rst

```diff
@@ -0,0 +1,37 @@
+.. doctest-skip-all
+
+************************************
+HITRAN Queries (`astroquery.hitran`)
+************************************
+
+Getting started
+===============
+
+This module provides an interface to the `HITRAN`_ database API.  It can
+download a data file including transitions for a particular molecule in a given
+wavenumber range.  The file is downloaded in the `~hitran.cache_location`
```

Comment:
You need to use the full path to make sphinx resolve it as an API link, e.g. ``~astroquery.hitran.cache_location``. Warnings for unresolvable links cause the build to fail.